acceptance-tests/build.gradle

```diff
@@ -17,7 +17,6 @@ dependencies {
   testRuntime 'org.apache.logging.log4j:log4j-core'
 
   testImplementation project(':ethsigner:core')
-
```

Comment:
nit: re-add just to remove this from the changeset.

Reply:
What is the best way to do that?